2007/6xxx/GSD-2007-6183.json

```diff
@@ -202,11 +202,15 @@
                 "CVE_data_version": "4.0",
                 "nodes": [
                     {
-                        "children": [],
+                        "children": [
+
+                        ],
```

Comment:
Oh, I thought my script did the opposite of this...

Okay, I don't like this, and will see if I can get the JSON serializer to stop making useless newlines.

Reply:
Fixed.